molecule/rbac-mtls-rhel-fips/molecule.yml

```diff
@@ -1,5 +1,5 @@
 ---
-### Installs Confluent Platform Cluster on RHEL8.
+### Installs Confluent Platform Cluster on Oracle Linux 9.
```

Comment:
In this scenario there is variable `redhat_java_package_name: java-11-openjdk` Should that line be removed as it wont affect oracle linux